### PR TITLE
feat: reduce rpc request

### PR DESCRIPTION
**Pull request by ghost**
This update aims to reduce rpc request in gateway, will combine separate `gatAccountInfo` to `getMultipleAccounts`, which is a helper function in `@dappio-wonderland/navigator`, for get multiple accounts at once.

### CODE REVIEW

File: src/protocols/raydium.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ import {
 } from "../ids";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { PoolInfoWrapper } from "@dappio-wonderland/navigator/dist/raydium";
+import { getMultipleAccounts } from "@dappio-wonderland/navigator/dist/utils";
```
Comment (ironaddicteddog) [inline src/protocols/raydium.ts]: This works for now. However imo `getMultipleAccounts` should be exported in `index.ts` from Navigator as well

File: src/protocols/raydium.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ import {
 } from "../ids";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { PoolInfoWrapper } from "@dappio-wonderland/navigator/dist/raydium";
```
Comment (ironaddicteddog) [inline src/protocols/raydium.ts]: This import is redundant. You can access `PoolInfoWrapper` via `raydium.PoolInfoWrapper`

File: src/protocols/saber.ts [hunk 3]
```diff
@@ -23,6 +23,7 @@ import {
 } from "../builder";
 import { SABER_ADAPTER_PROGRAM_ID, WSOL } from "../ids";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
+import { getMultipleAccounts } from "@dappio-wonderland/navigator/dist/utils";
```
Comment (ironaddicteddog) [inline src/protocols/saber.ts]: The as above

File: src/protocols/solend.ts [hunk 4]
```diff
@@ -15,6 +15,7 @@ import { IReserveInfo, solend } from "@dappio-wonderland/navigator";
 import { ActionType, GatewayParams, IProtocolMoneyMarket } from "../builder";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { NATIVE_SOL, SOLEND_ADAPTER_PROGRAM_ID, WSOL } from "../ids";
+import { getMultipleAccounts } from "@dappio-wonderland/navigator/dist/utils";
```
Comment (ironaddicteddog) [inline src/protocols/solend.ts]: Same as above

File: src/protocols/raydium.ts (hunk 1 above)
Comment (ironaddicteddog) [reply src/protocols/raydium.ts]: Try `utils. getMultipleAccounts`

Comment (ghost) [reply src/protocols/raydium.ts]: Sure!

File: package.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
     "testAdapterLifinity": "ANCHOR_WALLET=~/.config/solana/gateway.json ts-mocha -p ./tsconfig.json -t 1000000 tests/testAdapterLifinity.ts",
     "testAdapterOrca": "ANCHOR_WALLET=~/.config/solana/gateway.json ts-mocha -p ./tsconfig.json -t 1000000 tests/testAdapterOrca.ts",
     "testAdapterTulip": "ANCHOR_WALLET=~/.config/solana/gateway.json ts-mocha -p ./tsconfig.json -t 1000000 tests/testAdapterTulip.ts",
-    "testAdapterNftFinance": "ANCHOR_WALLET=~/.config/solana/gateway.json ts-mocha -p ./tsconfig.json -t 1000000 tests/testAdapterNftFinance.ts",
+    "testAdapterNftFinance": "ANCHOR_WALLET=~/.config/solana/dappio-test.json ts-mocha -p ./tsconfig.json -t 1000000 tests/testAdapterNftFinance.ts",
```
Comment (ironaddicteddog) [inline package.json]: Change to `gateway.json`

**Pull request merged**